resources/assets/js/annotations/components/annotationCanvas/polygonBrushInteraction.vue

```diff
@@ -99,32 +99,16 @@ export default {
                 this.map.addInteraction(shiftClickSelectInteraction);
             }
         },
+        toggleShiftClickSelectInteraction(mode) {
+            shiftClickSelectInteraction.setActive(this.canModify && (mode === 'polygonEraser' || mode === 'polygonFill'));
```

Comment:
This is probably better (even though I didn't do it myself in `toggleCurrentInteraction()`):

```suggestion
            shiftClickSelectInteraction.setActive(this.canModify && (this.isUsingPolygonEraser || this.isUsingPolygonFill));
```